source/extensions/outlier_detection_monitors/common/monitor_base_impl.cc

```diff
@@ -0,0 +1,79 @@
+#include "source/extensions/outlier_detection_monitors/common/monitor_base_impl.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Outlier {
+
+bool HTTPCodesBucket::match(const ExtResult& result) const {
+  // We should not get here with errors other then HTTP codes.
+  ASSERT(matchType(result));
+  const HttpCode& http_code = absl::get<HttpCode>(result);
```

Comment:
For little struct, actually value semantics would be better than reference. Like the Enum, int, bool, double, absl::string_view, etc.

Reply:
Sure. Will do.

Reply:
Done. Great suggestion. Thanks.